dir.targets

```diff
@@ -92,8 +92,8 @@
   <Import Project="$(MSBuildThisFileDirectory)illink.targets" />
 
   <ItemGroup Condition="'$(NuGetTargetMoniker)'=='.NETStandard,Version=v1.7'">
-    <!-- Temporarily suppress the message until we get a nuget version that knows about the mapping between netstandard1.7 and uap10.1 -->
-    <SuppressPackageTargetFrameworkCompatibility Include="uap10.1" />
+    <!-- Temporarily suppress the message until we get a nuget version that knows about the mapping between netstandard1.7 and uapvNext -->
```

Comment:
We do, @ericstj logged an issue some time ago, but we don't have the final moniker yet, so we don't want them to fix it just yet. This moniker we are going to use now will unblock @zamont 's testing, but it's not the final TFM to be used yet, we will still have to change it before shipping.